<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_main</class>
 <widget class="QWidget" name="ui_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f8f9fa;
    font-family: &quot;Roboto&quot;, &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, sans-serif;
    font-size: 14px;
    color: #202124;
}

QPushButton {
    background-color: #fff;
    color: #202124;
    border: 1px solid #dadce0;
    border-radius: 4px;
    padding: 10px 16px;
    font-size: 14px;
    font-weight: 500;
    margin-bottom: 8px;
}

QPushButton:hover {
    background-color: #e8f0fe;
    border-color: #e8f0fe;
}

QPushButton:pressed {
    background-color: #dbe9fd;
    border-color: #dbe9fd;
}

QPushButton:focus {
    outline: none;
    border-color: #1a73e8;
}
 
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTableView" name="tabla_Alumno"/>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="button_eliminar">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #ff9800; 
    color: #fff;
    border-color: #ff9800;
}

QPushButton:hover {
    background-color: #f57c00;
    border-color: #f57c00;
}

QPushButton:pressed {
    background-color: #e65100;
    border-color: #e65100;
}

QPushButton:focus {
    outline: none;
    border-color: #ff9800;
}</string>
        </property>
        <property name="text">
         <string>Eliminar Tabla</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_cargar">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: #1a73e8; 
    color: #fff;
    border-color: #1a73e8;
}

QPushButton:hover {
    background-color: #3c83f1;
    border-color: #3c83f1;
}

QPushButton:pressed {
    background-color: #185abc;
    border-color: #185abc;
}

QPushButton:focus {
    outline: none;
    border-color: #1a73e8;
}



</string>
        </property>
        <property name="text">
         <string>Cargar archivo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="butto_mostrar">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-color: #4caf50; 
    color: #fff;
    border-color: #4caf50;
}

QPushButton:hover {
    background-color: #43a047;
    border-color: #43a047;
}

QPushButton:pressed {
    background-color: #388e3c;
    border-color: #388e3c;
}

QPushButton:focus {
    outline: none;
    border-color: #4caf50;
}</string>
        </property>
        <property name="text">
         <string>Mostrar alumno</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>286</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="button_salir">
        <property name="styleSheet">
         <string notr="true">
QPushButton{
    background-color: #f44336; 
    color: #fff;
    border-color: #f44336;
}

QPushButton:hover {
    background-color: #e53935;
    border-color: #e53935;
}

QPushButton:pressed {
    background-color: #d32f2f;
    border-color: #d32f2f;
}

QPushButton:focus {
    outline: none;
    border-color: #f44336;
}</string>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
